src/cmd/flux-jobs.py

```diff
@@ -446,6 +468,8 @@ def format_field(self, value, spec):
         "nnodes": "NNODES",
         "ranks": "RANKS",
         "success": "SUCCESS",
+        "result": "RESULT",
+        "result_abbrev": "RESULT",
```

Comment:
Should heading for  abbreviated result string also be abbreviated? 
e.g.

```python
        "result_abbrev": "RS",
```

or something.

Reply:
seems like a good idea.  And I suppose "STATE" should be "ST", I'll go ahead and change both in a new commit.